tests/NUnit.OneTimeSetup.DreddLogs.Tests.Internal/WithoutGlobalSetup/SyncSetup/TestFixtureWithSynchronousNonStaticSetupMethod.cs

```diff
@@ -11,11 +11,16 @@ public class TestFixtureWithSynchronousNonStaticSetupMethod : BaseTest
         public void OneTimeSetup()
         {
             Logger.Information($"{nameof(OneTimeSetup)} of {nameof(TestFixtureWithSynchronousNonStaticSetupMethod)}");
-            throw new Exception("Synchronous non-static fixture setup method");
+            ThrowException();
         }
 
         [Test]
         public void Test()
         { }
+
+        private void ThrowException()
```

Comment:
What the main point to extract this method?

Reply:
The purpose was to test that stacktrace is assembled in correct way.